### ISSUE TITLE
New release not triggered

### ISSUE THREAD

**Issue by bhartesh77** [mfrachet/rn-placeholder#226]
@mfrachet I see the PR to change peer dependencies was merged, but a new release was not triggered, please have a look at it when you get the time.

PR: https://github.com/mfrachet/rn-placeholder/pull/212
CC: @brian-perlego

**Comment by ericlifs** [comment]
Same issue here, can you please create a new release @mfrachet?

Thanks in advance!

**Comment by Ahmedhamed77** [comment]
same issue here

**Comment by Ahmedhamed77** [comment]
is this package still maintained?

**Comment by bhartesh77** [comment]
@Ahmedhamed77 I don't think so, but you can fork the project. Then update the react-native version, and install this new forked package from the github url.

**Comment by fzxlzy** [comment]
Same issue here, can you please create a new release @mfrachet?

Thanks in advance!
